jflex/src/main/java/jflex/NFA.java

```diff
@@ -149,9 +155,9 @@ public void addStandaloneRule() {
    */
   public void addRegExp(int regExpNum) {
 
-    if (Options.DEBUG)
-      Out.debug(
-          "Adding nfa for regexp " + regExpNum + " :" + Out.NL + regExps.getRegExp(regExpNum));
+    if (GeneratorOptions.DEBUG)
```

Comment:
This is good. I didn't first see that `GeneratorOptions.DEBUG` is still around. The other parts that used `Options.DEBUG` should use this one, and the other smaller scope static `DEBUG`s can just stay in the classes they are (they are separate either because they slow things down a lot or produce huge amounts of output unless you are looking at specific test cases).

Reply:
This one was easy to keep because the NFA already depends on the Out instance. I'm afraid the others were not, and I didn't want to bring a dep on Out just for a DEBUG statement.